.travis.yml

```diff
@@ -8,9 +8,9 @@ python:
 
 before_install:
   - sudo apt-get update
-  - sudo apt-get install -y graphviz
+  - sudo apt-get install -y graphviz plantuml
 
-install: pip install sphinx sphinx_rtd_theme
+install: pip install sphinx==3.5.4 sphinx_rtd_theme sphinxcontrib-plantuml
```

Comment:
try:
`pip install sphinx==1.8.5 sphinx_rtd_theme sphinxcontrib-plantuml Pillow`
Using sphinx 3.x shall fix the build issue. Pillow is needed to fix scaling of generated images (WARNING: plantuml: unsupported scaling attributes: width (install PIL or Pillow)).
Tested in my Linux Mint 20 VM.